spec/controllers/queue_items_controller_spec.rb

```diff
@@ -78,101 +82,158 @@
       delete :destroy, id: queue_item.id
       expect(response).to redirect_to my_queue_path
     end
-    
+
     it "deletes the queue item" do
       user1 = Fabricate(:user)
       set_current_user(user1)
       queue_item = Fabricate(:queue_item, user: user1)
       delete :destroy, id: queue_item.id
-      expect(QueueItem.count).to eq(0)      
+      expect(QueueItem.count).to eq(0)
     end
-    
+
     it "normalizes the remaining queue item positions" do
       user1 = Fabricate(:user)
       set_current_user(user1)
-      queue_item1 = Fabricate(:queue_item, user: user1, position: 1)
-      queue_item2 = Fabricate(:queue_item, user: user1, position: 2)
+      queue_item1 = Fabricate(
+        :queue_item,
+        user: user1,
+        position: 1
+      )
+      queue_item2 = Fabricate(
+        :queue_item,
+        user: user1,
+        position: 2
+      )
       delete :destroy, id: queue_item1.id
       expect(QueueItem.first.position).to eq(1)
     end
-    
-    it "does not delete the queue item if the queue item is not in the current user's queue" do
+
+    it "does not delete the queue item if it is not in the current user's queue" do
       user1 = Fabricate(:user)
       user2 = Fabricate(:user)
       set_current_user(user1)
       queue_item_user2 = Fabricate(:queue_item, user: user2)
       delete :destroy, id: queue_item_user2.id
-      expect(QueueItem.count).to eq(1) 
+      expect(QueueItem.count).to eq(1)
     end
-    
+
     it_behaves_like "requires sign in" do
       let(:action) { delete :destroy, id: 1 }
     end
   end
-  
+
   describe "POST update_queue" do
     it_behaves_like "requires sign in" do
-      let(:action) { post :update_queue, queue_items: [{id: 1, position: 2}, {id: 2, position: 1}] }
+      let(:action) do
+        post :update_queue, queue_items: [
+          { id: 1, position: 2 },
+          { id: 2, position: 1 }
+        ]
+      end
     end
+
     context "with valid inputs" do
       let(:user1) { Fabricate(:user) }
       let(:video) { Fabricate(:video) }
-      let(:queue_item1) { Fabricate(:queue_item, user: user1, position: 1, video: video) }
-      let(:queue_item2) { Fabricate(:queue_item, user: user1, position: 2, video: video) }
-    
+      let(:queue_item1) { Fabricate(:queue_item,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.